README.md

```diff
@@ -1,39 +1,82 @@
+- **They handle loading and error states for you.** No need to sweat the details of handling the different states. We've done that for you.
 
-With your metric and access token in hand, it's time to start using Propel UI Kit. Currently, the UI Kit is available for React only and supports the following metric components:
+- **They fetch the data for you.** Components can automatically fetch the data from the Propel API when used in connected mode.
+
+Individual components have other key features that we highlight on the component documentation pages.
+
+## Components
+
+Check out the different components and their documentation:
 
 - [Time Series](https://github.com/propeldata/ui-kit/tree/main/packages/react/time-series)
 - [Counter](https://github.com/propeldata/ui-kit/tree/main/packages/react/counter)
 - [Leaderboard](https://github.com/propeldata/ui-kit/tree/main/packages/react/leaderboard)
 
-Now let's understand how you can integrate Propel UI Kit with your front-end application.
+Please [open a feature request](https://github.com/propeldata/ui-kit/issues) on GitHub for other components you'd like to have. Also, PRs are welcome 😊.
+
+## Getting Started with Propel
+
+To leverage the full power of Propel UI Kit + Propel's API, you'll need a Propel Account. If you don't have a Propel Account yet, you can [try Propel for free](https://propeldata.com/get-started).
+
+You can learn more about Propel by reading our [API docs](https://propeldata.com/docs).
+
+Once you have your account set up, you'll need to provide your Account's access token to use Propel APIs. You can authenticate and generate an access token by following the steps outlined in [the Propel GraphQL Authentication guide](https://propeldata.com/docs/api/authentication#step-2-generate-an-access-token).
 
 ## Usage
 
-The components in the Propel UI Kit are available in two modes: "Static" and "Connected". The "Static" mode is a simpler version of the component, suitable for users who want to quickly build basic visualizations or handle server data themselves. The "Connected" mode is a more advanced version of the component, which offers out-of-the-box integration with propel APIs. You don't need to configure a GraphQL client, handle loading and error states. You don't even need to know what the Propel API endpoint is.
+The Propel UI Kit components are available in two modes: "Static" and "Connected".
+
+In the "Static" mode, you fetch the data yourself and provide it to the component.
 
-Overall, the Propel UI Kit is a powerful tool for front-end developers who want to create engaging, user-friendly data visualizations with ease.
+In the "Connected" mode, the component automatically fetches the data from the Propel API. You don't need a backend, to configure a GraphQL client, or handle loading and error states.
```

Comment:
Nit: this is kind of redundant. Very similar information is re-presented under the `### Static Mode` and `### Connected Mode` sections. Tighten things up and get rid of these?